boards/samr21-xpro/Makefile.dep

```diff
@@ -1,4 +1,4 @@
 ifneq (,$(filter gnrc_netif_default,$(USEMODULE)))
   USEMODULE += at86rf233
-  USEMODULE += gnrc_nomac
+  USEMODULE += gnrc_lwmac
```

Comment:
This is not meant to be merged as-is. There is intentionally a separate commit that replaces nomac and eases testing. Since there has only been one MAC layer, I didn't know how to integrate it properly, so I just replaced it.